assets/blocks/course-outline/module-block/module-block.scss

```diff
@@ -100,7 +100,7 @@
 
 	.block-editor-block-styles__item & {
 		font-size: 150%;
-		.block-editor-inner-blocks, #{$block}__progress-indicator, .wp-block-sensei-lms-course-outline__arrow {
```

Comment:
@yscik, I noticed the inner-blocks were hidden [here](https://github.com/Automattic/sensei/commit/353715f5bf0a4ab531a81ed00a3760aa96298344). If you can take a look, please let me know if I missed/broke something. 😉